solidity/contracts/converter/ConverterFactory.sol

```diff
@@ -15,15 +16,28 @@ contract BancorConverterFactory is IBancorConverterFactory {
     */
     event NewConverter(address indexed _converter, address indexed _owner);
 
+    mapping (uint8 => ITypedConverterFactory) public factories;
+
     /**
-      * @dev initializes a new BancorConverterFactory instance
+      * @dev initializes a new ConverterFactory instance
     */
     constructor() public {
     }
 
+    /**
+      * @dev initializes the factory with a specific typed factory
+      * can only be called by the owner
+      * 
+      * @param _type              converter type, see BancorConverter contract main doc
+      * @param _factory           typed factory
+    */
+    function setTypedConverterFactory(uint8 _type, ITypedConverterFactory _factory) public ownerOnly {

```

Comment:
@yudilevi : IMO, interface `ITypedConverterFactory` should expose a function called `factoryType`, which we should then use here instead of passing `_type`.

Reply:
added converterType() function